goes16_benchmark_casper.sh

```diff
@@ -2,15 +2,11 @@
 #SBATCH --job-name=goes16ci
 #SBATCH --account=NAML0001
 #SBATCH --ntasks=32
-#SBATCH --time=01:00:00
+#SBATCH --time=04:00:00
 #SBATCH --partition=dav
 #SBATCH --mem=128G
 #SBATCH --cpus-per-task=1
 #SBATCH --gres=gpu:v100:1
 module purge
```

Comment:
Can you remove the module purge and add module load cuda/10.1 back in? It will have trouble training otherwise.